jvm/src/main/scala/async/VThreadSupport.scala

```diff
@@ -13,11 +14,14 @@ object VThreadScheduler extends Scheduler:
   override def execute(body: Runnable): Unit = VTFactory.newThread(body)
 
   override def schedule(delay: FiniteDuration, body: Runnable): Cancellable =
+    val interruptGuard = AtomicBoolean(true) // to avoid interrupting the body
```

Comment:
Great but use varHandle to reduce memory usage？

Reply:
@He-Pin can you elaborate?

Reply:
Like atomic interger field updater, which can reduce the 4byte ref of the object wrapper.